docker-compose.yml

```diff
@@ -1,3 +1,17 @@
+# Copyright 2022, OpenTelemetry Authors
```

Comment:
Remove the year, it's hard to maintain and adds little if no value.

https://github.com/open-telemetry/opentelemetry-java/blob/19da306e9aefa35b79314d4876e319a743566b95/api/all/src/jmh/java/io/opentelemetry/api/baggage/propagation/W3CBaggagePropagatorBenchmark.java#L2